app/src/lib/git/core.ts

```diff
@@ -101,6 +101,10 @@ export async function git(args: string[], path: string, name: string, options?:
   const defaultOptions: IGitExecutionOptions = {
     successExitCodes: new Set([ 0 ]),
     expectedErrors: new Set(),
+    env: {
+      GIT_TERMINAL_PROMPT: '0',
```

Comment:
You're right. I assumed it was necessary as it's used for every git command in [atom/github](https://github.com/atom/github/blob/master/lib/git-shell-out-strategy.js#L78). I removed the line in 54cee14.